BlockSettleUILib/Trading/RFQReplyWidget.ui

```diff
@@ -1,14 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```

Comment:
Restore copyright text

Reply:
this is auto generated file, so it will always delete this header. Maybe we should run script afterwards in some period of time which is adding this header?

Reply:
Maybe you should do it manually now?

Reply:
I will, thank you

Reply:
Thank you